manageiq-operator/deploy/role.yaml

```diff
@@ -2,57 +2,79 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   creationTimestamp: null
-  name: manageiq-operator
+  name: manageiq-operator/
 rules:
 - apiGroups:
   - ""
   resources:
   - pods
   - services
+  - services/finalizers
+  - endpoints
```

Comment:
Please don't change any of these unless it's necessary.
https://github.com/ManageIQ/manageiq-pods/pull/413 set the roles that we need. I don't see why updating the sdk version would change any of this.

Reply:
This was generated by `operator-sdk`, can we keep this?

Reply:
No, this clearly should be edited to suit what is required for the individual application. The sdk generator is a guideline for starting a project. As the project owners we know what roles we require and should not request any that we do not need.

Reply:
Done